locales/en/breaches.ftl

```diff
@@ -2,6 +2,24 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+breach-meta-title = { -brand-fx-monitor } - Dashboard
+
+breach-all-meta-title = { -brand-fx-monitor } - All Data Breaches
+breach-all-meta-social-title = All Breaches Detected by { -brand-fx-monitor }
+breach-all-meta-social-description = Browse the complete list of known breaches detected by { -brand-fx-monitor }, then find out if your information was exposed.
+
+# Variables:
+#   $company (String) - Name of the company that was breached, e.g. "PHP Freaks"
+breach-detail-meta-title = { -brand-fx-monitor } - { $company } Data Breach
+# Variables:
+#   $company (String) - Name of the company that was breached, e.g. "PHP Freaks"
+breach-detail-meta-social-title = Were you affected by the { $company } Data Breach?
+breach-detail-meta-social-description = Use { -brand-fx-monitor } to find out if your personal information was exposed in this breach, and understand what to do next.
+
+breach-scan-meta-title = { -brand-fx-monitor } - Breach Results
+breach-scan-meta-social-title = Firefox Monitor Breach Results
```

Comment:
These two strings need to use the Fluent term instead of hard-coding the brand name.

Reply:
Sorry, I was in the middle of making changes and pushed to investigate a test failure we had issues with that wasn’t related to the strings. I was planning on re-requesting a review when ready.

We are currently in the middle of process changes and in a phase where there are some open ends. We found improvements and additions that we needed to do — sorry for the inconvenience.